spec/helper.js

```diff
@@ -21,6 +21,27 @@ global.onHttpContext = index.onHttpContextFactory();
 // Legacy
 global.dihelper = onHttpContext.helper;
 
+helper.httpServerBefore = function(overrides, endpointOpt) {
+    before('helper.httpServer.before', function() {
+        this.timeout(5000);
```

Comment:
I see that some cases are setting timeout to 10000ms, if all are set to 5000ms, is it sure that it doesn't have random issues?